include/lsst/afw/detection/Psf.h

```diff
@@ -254,14 +299,30 @@ class Psf : public daf::base::Citizen,
                                                       std::string const& warpAlgorithm = "lanczos5",
                                                       unsigned int warpBuffer = 5);
 
+    /** Return the capacity of the caches
+     *
+     * Both the image and kernel image caches have the same capacity.
+     */
+    std::size_t getCacheCapacity() const { return _kernelImageCache->capacity(); }
+
+    /** Set the capacity of the caches
+     *
+     * Both the image and kernel image caches will be set to this capacity.
+     */
+    void setCacheCapacity(std::size_t capacity) {
+        _imageCache->reserve(capacity);
+        _kernelImageCache->reserve(capacity);
+    }
```

Comment:
If you define `setCacheCapacity` and `getCacheCapacity` (and `~Psf()`, I think), you should no longer need to include `Cache.h` in other source files that use `Psf`.  I'm not totally sure that's enough, but the goal here is to make sure that anything that uses `Cache` as more than an opaque pointer is in `Psf.cc`, and that will make it unnecessary to include `Cache.h` anywhere but `Psf.cc`.